Bio/Cluster/clustermodule.c

```diff
@@ -2318,7 +2320,7 @@ static char pca__doc__[] =
 "Adding the column means to the dot product of the coordinates and the\n"
 "principal components, i.e.\n"
 "\n"
-"    >>> columnmean + dot(coordinates, pc)\n"
+"   'columnmean + dot(coordinates, pc)'\n"
```

Comment:
Do we need the single quotes here?

Reply:
I think `>>>` was used to give it nice source code formatting, but that triggered a doctest. So I thought at least a single quote would be better than nothing. Feel free to add a commit that uses that preferred formatting for non-tested code!

Reply:
I was just thinking that the quotes make it look like a string. If you remove the single quotes, I am OK with this pull request.